releasenotes/notes/add-examples-to-experiments-api-docs-31f3e3c3369a6f3d.yaml

```
@@ -0,0 +1,32 @@
+---
+other:
+  - |
+    Added minimal working code examples to the API pages for Experiments,
+    EFSpectroscopy,
```

Could you make all of these references?

```suggestion
    :class:`~.EFSpectroscopy`,
```

---

done